.github/workflows/rust.yml

```diff
@@ -47,7 +47,13 @@ jobs:
     - run: rustup update
     - run: rustup target add ${{ matrix.target }}
     - run: cargo clippy --all-targets --target ${{ matrix.target }}
-    - run: cargo test --target ${{ matrix.target }}
+    # unit tests
+    - run: cargo test --bins --target ${{ matrix.target }}
+    # integration tests
+    # TODO: we should probably manage the container in the test itself
+    # TODO: do a proper functional check on the container startup instead of waiting 15 seconds
+    - run: docker run -d -p 8000:8000 stellar/quickstart:soroban-dev --standalone --enable-soroban-rpc --protocol-version 20 && sleep 15
```

Comment:
If we're going to use the quickstart image in integration/e2e tests, we should enable artificial acceleration mode. You can enable it by appending this option: `--enable-core-artificially-accelerate-time-for-testing`.

(Artificial acceleration made a big difference to the tests of the Starlight project where we do the same thing: https://github.com/stellar/starlight/blob/main/.github/workflows/sdk.yml#L59-L62.)

Reply:
I will add it